robot/resources/lib/python_keywords/object_access.py

```diff
@@ -142,6 +143,12 @@ def can_get_head_object(
     return True
 
 
+def _generate_random_range_cut(start: int = 0, end: int = 10):
+    start = random.randint(start, end)
+    end = random.randint(start, end)
+    return f"{start}:{end}"
```

Comment:
What if it's 10:10 or 0:0?

Reply:
I assumed these are valid ranges, at least the command works:

```
COMMAND: neofs-cli --config /home/runner/work/neofs-node/neofs-node/neofs-testcases/wallet_config.yml object range --rpc-endpoint 's01.neofs.devenv:8080' --wallet '/home/runner/work/neofs-node/neofs-node/neofs-testcases/TemporaryDir/9565822e-6a8a-4dd6-8191-d1e79649b0bd.json' --cid '61M7pv5kkku8yNhrLiohmLd31XuSDCR3WLQkiGrd4Sgd' --oid 'B5vP6r1BQ3ktxY2wfvDNwBy6UnKYTRAQRCCqcCzUgGsb' --range '10:10' --file 'TemporaryDir/TestObjectsDir/3bbbe60d-6fdd-4b04-9449-0fd2e8826521'
RETCODE: 0

STDOUT:
[TemporaryDir/TestObjectsDir/3bbbe60d-6fdd-4b04-9449-0fd2e8826521] Payload successfully saved

STDERR:

Start / End / Elapsed	 19:31:43.302994 / 19:31:43.707277 / 0:00:00.404283
```

Reply:
fixed according to Pavel's comment